international-review-of-the-red-cross.csl

```diff
@@ -195,13 +188,27 @@
   <citation et-al-min="4" et-al-use-first="1" disambiguate-add-names="true">
     <layout suffix="." delimiter="; ">
       <choose>
-        <if position="subsequent">
+        <if match="any" position="ibid ibid-with-locator">
```

Comment:
are you using this construction on purpose? What this does as opposed to separately going through ibid and ibid-with-locator is to add the page number even were you're referring to the same one. I.e. this code produces
1. Smith, p. 59
2. Ibid. p. 59

that's unusual and I don't see anything in the guide to justify it.